### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by Tarrasch**
There have been quite a lot of issues recently. By being explicit upfront. We can be less emotional and subjective when closing issues in the future. https://github.com/spotify/luigi/issues/2029 https://github.com/spotify/luigi/issues/2028 https://github.com/spotify/luigi/issues/2027

### CODE REVIEW

**Comment by dlstadther** [comment]
This is fine with me. The travis build didn't finish, but this commit won't break anything as it only adds a template.

**Comment by henryrizzi** [comment]
Mind if I clean up some spelling and wording things for clarity?

**Comment by dlstadther** [comment]
The clearer, the better. @Tarrasch is spending less time on Luigi now-a-days. So if you'd like to submit a cleaner PR for a `ISSUE_TEMPLATE.md`, feel free.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!---
+There's currently no strict procedure or guidelines to submitting issues.
```
Comment (henryrizzi) [inline ISSUE_TEMPLATE.md]: This sounds better I think:
>Currently, there are no strict procedures or guidelines for submitting issues.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Briefuly, just use common sense.
```
Comment (henryrizzi) [inline ISSUE_TEMPLATE.md]: > In brief, please just use common sense.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+Common sense at least includes:
```
Comment (henryrizzi) [inline ISSUE_TEMPLATE.md]: >Common sense includes this at bare-minimum:

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+ * search for similar issues posted before creating a new issue.
+ * Use markdown to format all code/logs. Issues which are hard to read
+   when rendered on GitHub might be closed with a friendly reminder of this.
+
+Also, if applicacable add steps to reproduce a bug. And feel free to include sample code.
```
Comment (henryrizzi) [inline ISSUE_TEMPLATE.md]: > Also, add steps to reproduce the bug if applicable. And sample code if you'd like. :)

**Comment by dlstadther** [comment]
Maybe it would be good to direct Issue and PR templates to [this](https://opensource.guide/how-to-contribute/#how-to-submit-a-contribution) opensource contributions guide as a (`for more information on how to submit valuable contributions, see https://opensource.guide/how-to-contribute/#how-to-submit-a-contribution`).

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (dlstadther) [inline ISSUE_TEMPLATE.md]: `In brief` sounds weird. Perhaps `In short, please just use common sense`

File: ISSUE_TEMPLATE.md (hunk 4 above)
Comment (dlstadther) [inline ISSUE_TEMPLATE.md]: `also, add steps to reproduce the bug, if applicable. Sample code would be nice too :)`

**Comment by Tarrasch** [comment]
Thanks for your valuable feed back guys!

@dlstadther, that's a fantastic link! I'll be sharing it with my friends too! :)

**Pull request merged**